### PR TITLE
Brush up

### PR DESCRIPTION
**Pull request by uu59**
- Add `rake gemfiles` task to test this plugin on multiple Embulk versions.
- Remove old Embulk (< 0.6.12) compatible code

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -8,6 +8,40 @@ task :test do
   ruby("test/run-test.rb", "--use-color=yes")
 end
 
+desc "Generate gemfiles to test this plugin with released Embulk versions (since MIN_VER)"
+task :gemfiles do
+  min_ver = Gem::Version.new(ENV["MIN_VER"] || "0.6.12")
+  puts "Generate Embulk gemfiles from #{min_ver} to latest"
+
+  embulk_tags = JSON.parse(open("https://api.github.com/repos/embulk/embulk/tags").read)
+  embulk_versons = embulk_tags.map{|tag| Gem::Version.new(tag["name"][/v(.*)/, 1])}
```
Comment (yoshihara) [inline Rakefile]: s/verson/version/ ?

Comment (uu59) [inline Rakefile]: :palm_tree:

**Comment by yoshihara** [comment]
LGTM

**Comment by yoshihara** [comment]
LGTM

**Pull request merged**